scripts/cluster_providers/kind_setup_cluster

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
```

Comment:
the script should receive kube version as argument, check if that specific `KIND_xxx` env var exist, and if not create cluster under the given name. In general the script should implement this part of prev logic https://github.com/helm/acceptance-testing/blob/master/lib/Kind.py#L42-L46 as well.

Reply:
return cluster_name from create_cluster script